cmd/skaffold/app/cmd/inspect_tests.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"context"
+	"io"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/inspect"
+	tests "github.com/GoogleContainerTools/skaffold/pkg/skaffold/inspect/tests"
+)
+
+func cmdTests() *cobra.Command {
+	return NewCmd("tests").
+		WithDescription("Interact with configuration tests").
+		WithPersistentFlagAdder(cmdTestsFlags).
+		WithCommands(cmdTestsList())
+}
+
+func cmdTestsList() *cobra.Command {
+	return NewCmd("list").
+		WithExample("Get list of tests", "inspect tests list --format json").
+		WithExample("Get list of tests targeting GoogleCloudBuild environment", "inspect tests list --build-env googleCloudBuild --format json").
+		WithDescription("Print the list of profile names that can be invoked with the --profile flag in other skaffold commands.").
```

Comment:
description seems incorrect for the command.

Reply:
also this could use a `--profile` flag similar to the `--module` flag?

Reply:
fixed both, added better description (forgot this sorry) and added `--profile` flag w/ similar description to buildEnv (where `--profile` flag is also used)